host/app/lib/card-api.gts

```diff
@@ -20,6 +21,14 @@ type FieldsTypeFor<T extends Card> = {
 
 type Setter = { setters: { [fieldName: string]: Setter }} & ((value: any) => void);
 
+interface ResourceObject {
+  // id: string; // TODO
+  type: string;
```

Comment:
so currently over on the compiler side, we are using using the type `card`. I think it's fine to hard code for now.